src/main/java/io/jenkins/plugins/coverage/targets/CoverageElement.java

```diff
@@ -29,7 +29,7 @@
  * @since 22-Aug-2007 18:36:01
  */
 public enum CoverageElement {
-    AGGREGATED_REPORT(()->"Aggregated Report"),
+    AGGREGATED_REPORT(() -> "Aggregated Report"),
```

Comment:
As we discussed at the call, maybe it's better to convert it to "private final String name" instead of using Lambdas. But maybe there are some extensibility plans for the future.  CC @jeffpearce